From d4132f043f71917b6d1fb0f0063edbe96f0b4759 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:49:09 -0400
Subject: [fs] nfsd: fix Kconfig syntax

Message-id: <1403560314-32525-8-git-send-email-bfields@redhat.com>
Patchwork-id: 84448
O-Subject: [Patch RHEL7 007/172] nfsd: fix Kconfig syntax
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Christoph Hellwig <hch@infradead.org>

The description text for CONFIG_NFSD_V4_SECURITY_LABEL has an unpaired
quote sign which breaks syntax highlighting for the nfsd Kconfig file.
Remove it.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: cce6de908ee934fbea12d6cccc55b87dbd4b0c8a
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/Kconfig b/fs/nfsd/Kconfig
index dc8f1ef..f994e75 100644
--- a/fs/nfsd/Kconfig
+++ b/fs/nfsd/Kconfig
@@ -95,7 +95,7 @@ config NFSD_V4_SECURITY_LABEL
  Smack policies on NFSv4 files, say N.
 
  WARNING: there is still a chance of backwards-incompatible protocol changes.
- For now we recommend "Y" only for developers and testers."
+ For now we recommend "Y" only for developers and testers.
 
 config NFSD_FAULT_INJECTION
  bool "NFS server manual fault injection"
-- 
1.7.1